src/index.js

```diff
@@ -34,11 +35,14 @@ export function call(obj, args, cons = console){
     }
 }
 
-export function run(cmd, options){
-    options = options || {};
+export function run(cmd, options = {}){
+    options.env  = options.env || {};
+    options.env.PATH = [
+      `${process.cwd()}/node_modules/.bin/`, process.env.PATH 
+    ].join(path.delimiter);
```

Comment:
That also wouldn't work. `path.delimiter` is the character that particular OSes use to separate out their paths. So it's `:` for unix and `;` for windows.